aws/credentials/stscreds/assume_role_provider.go

```diff
@@ -255,8 +271,9 @@ func (p *AssumeRoleProvider) Retrieve() (credentials.Value, error) {
 		// Expire as often as AWS permits.
 		p.Duration = DefaultDuration
 	}
+	jitter := time.Duration(sdkrand.SeededRand.Int63n(int64(p.MaxJitter)))
```

Comment:
I think we'll need to make minor change here, the `Int63n` method cannot be called with a zero value parameter.  (Noticed this in the CI test failures)

suggest changing this to somethin glike:

```go
var jitter time.Duration
if p.MaxJitter > 0 {
    jitter = time.Duration(sdkrand.SeededRand.Int63n(int64(p.MaxJitter)))
}
```